src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Utilities/ReferenceCountedDisposable.cs

```diff
@@ -225,11 +224,7 @@ public ValueTask DisposeAsync()
         /// </summary>
         public struct WeakReference
```

Comment:
i def thing we need to doc this guy *Strongly*.

Reply:
➡️ Documented and tested